### ISSUE TITLE
ELMo embedding to return numpy array

### ISSUE THREAD

**Issue by jtizzle36** [allenai/allennlp#1355]
In the tutorial it states to use the allennlp.commands.elmo.ElmoEmbedder to return numpy arrays instead of a torch tensor. It is unclear to me how to do this as part of a script. Does the instruction mean to use the CLI approach rather than using the allennlp library as part of a script?

**Comment by nelson-liu** [comment]
Yes, `allennlp elmo` is a CLI interface to `allennlp.commands.elmo.ElmoEmbedder`.
